src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -153,13 +153,13 @@ protected override void StopProcessing()
         /// <summary>
         /// Run a command.
         /// </summary>
-        protected void RunCommand(String command, String args) {
+        protected void RunCommand(string command, string args) {
```

Comment:
The way `/sbin/shutdown` is invoked makes it impossible to detect failure on the PowerShell side, though that's probably for a separate PR; see

* #19802